### PR TITLE
Function for setting text appearance of Done and Skip buttons

### PR DESCRIPTION
**Pull request by bmalex88**
This time the two functions allow changing the entire appearance of the Done and Skip buttons

### CODE REVIEW

**Comment by bmalex88** [comment]
I also found an issue after increasing the text size for **DONE** and **SKIP**. The arrows are no longer vertically centered if I do so.

bottom-displaced

Inexplicably somehow, if the style for the **DONE** and **SKIP** is separated from the the style of the **NEXT** and **BACK** buttons, everything is fixed.

bottom-centered

This also allows for separate customization of the _text buttons_ from the _image views buttons_.

**Comment by cortinico** [comment]
> I also found an issue after increasing the text size for **DONE** and **SKIP**. The arrows are no longer vertically centered if I do so.

Thank you very much for your findings @bmalex88 
Can I ask you to separate this in another PR (as one is technically a bugfix and the other is a new feature).

File: appintro/src/main/res/layout/appintro_intro_layout.xml [hunk 1]
```diff
@@ -33,6 +33,7 @@
         android:id="@+id/bottom"
         android:layout_width="match_parent"
         android:layout_height="@dimen/appintro_bottombar_height"
+        android:padding="0dp"
```
Comment (cortinico) [inline appintro/src/main/res/layout/appintro_intro_layout.xml:36]: Why is this needed?

**Review by cortinico — APPROVED** [review]
LGTM 👌  Thank you very much for the contrib
